cev_eris.dme

```diff
@@ -2149,11 +2151,10 @@
 #include "code\modules\projectiles\_defines.dm"
 #include "code\modules\projectiles\ammunition.dm"
 #include "code\modules\projectiles\dynamic_cursor.dm"
-#include "code\modules\projectiles\effects.dm"
 #include "code\modules\projectiles\gun.dm"
 #include "code\modules\projectiles\gun_firemode.dm"
 #include "code\modules\projectiles\gun_hud_actions.dm"
-#include "code\modules\projectiles\projectile.dm"
+#include "code\modules\projectiles\projectile_new.dm"

```

Comment:
I left the original file there to help with debugging. It can be removed before this is merged.